support-models/src/main/scala/com/gu/support/workers/PaymentFields.scala

```diff
@@ -32,14 +33,38 @@ object StripePublicKey {
     else Left(s"StripePublicKey <$value> had invalid characters")
 
   def get(value: String): StripePublicKey =
-    parse(value).left.map(new Throwable(_)).toTry.get
+    parse(value).left.map(new Throwable(_)).toTry.get // Try.get will rethrow the useful error
 
   implicit val decoder: Decoder[StripePublicKey] = Decoder.decodeString.emap(parse)
   implicit val encoder: Encoder[StripePublicKey] = Encoder.encodeString.contramap[StripePublicKey](_.value)
 
 }
 case class StripePublicKey private (value: String) extends AnyVal
 
+object StripeSecretKey {
```

Comment:
I've also added a wrapper for the secret key, this means it is also parsed at the point of loading and checked that the PK and SK are the right way around and formatted correctly.  Then the type system will ensure that they are used correctly after that.